add-projects-repos-to-l3af.md

```diff
@@ -0,0 +1,18 @@
+# Adding projects/repositories to the l3af-project organisation
+
+This document outlines the process for adding a new project to the [l3af-project GitHub organisation](https://github.com/l3af-project). This is intended to be an initial and lightweight process which can be changed and expanded later as needed. If changes _are_ needed, please [open an issue in the governance repo](https://github.com/l3af-project/governance/issues/new) then add the issue to the agenda for a TSC meeting.
+
+## Definitions, constraints, etc.
+
+For the sake of this document, "project" will be synonymous with "git repository" or "git repositories" if required by the project. "Project", "repo", and "repository" will be used interchangeably.
+
+All repositories under the [l3af-project GitHub organisation](https://github.com/l3af-project) must be related in some way to the L3AF project.
+
+## Process
+
+All L3AF repos must be approved and added by the Technical Steering Committee (TSC). Here's how we go about doing that.
+
+1. Ask TSC for repo
```

Comment:
Maybe add "Request TSC to add the repo and state the reason for adding it and why it is needed"